### PR TITLE
dash and comma

### PR DESCRIPTION
**Pull request by Volvo-hackangel**

### CODE REVIEW

File: script/Arcueid/Day 13/13_00_ARC13_4.txt [hunk 1]
```diff
@@ -1266,7 +1266,8 @@ Time had already come to an end the instant she delivered that blow against Roa.
 [sha:14bbb8d33a88428f902443ffa84bfb9af473f42b]{
 -- Page 382, Offset 32734.
 -- 「―――、―――」
-"―――, ―――"
+// unvoiced
+"――――――"
```
Comment (rschlaikjer) [inline script/Arcueid/Day 13/13_00_ARC13_4.txt:1270]: Unvoiced nara single quotes

Comment (rschlaikjer) [reply script/Arcueid/Day 13/13_00_ARC13_4.txt:1270]: Eh actually sure whatever he's voicing silence

File: script/Arcueid/Day 5/05_00_ARC05_8.txt [hunk 2]
```diff
@@ -88,7 +88,7 @@ The white beauty in front of him is swallowed completely.
 [sha:9c453a2810079dd648098657d6d38fc360da3040]{
 -- Page 522, Offset 9997.
 -- 「――――――、え？」
-"――――――, Eh?"
+"――――――Eh?"
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_8.txt:91]: We can consider changing the `Eh`s to `Huh`s to appease the nitpickers who don't understand that Shiki is just Canadian

Comment (Volvo-hackangel) [reply script/Arcueid/Day 5/05_00_ARC05_8.txt:91]: I can do that if you want. We already localized the fufufus anyway

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 6/06_00_ARC06_1_3.txt [hunk 3]
```diff
@@ -103,15 +103,15 @@ I had thought for sure that you would have reflected on your abysmal behavior th
 // Akiha -> your since he's talking straight at her
 But that has no bearing on my good mood. I'm just happy because I get to see your face first thing in the morning."
 }
-[sha:0d4898c497e1af6b4e2ecfcf7c11a1d1126b20e4]{
+[offset:14211]{
```
Comment (rschlaikjer) [inline script/Arcueid/Day 6/06_00_ARC06_1_3.txt:106]: Why keep this one?

File: script/Ciel/Day 9/09_01_CIEL09_4.txt [hunk 4]
```diff
@@ -642,7 +642,7 @@ She's a butcher that shows no concern about other people's lives, who's neverthe
 -- それが俺に聞かせたかった本題らしい。
 That seems to be what she wanted me to hear the most.
 }
-[sha:670420aaa3cb10cfc520fd1f52aebab89524bae4]{
+[offset:23232]{
```
Comment (rschlaikjer) [inline script/Ciel/Day 9/09_01_CIEL09_4.txt:645]: Why are all of these now offsets if they're still the same?

File: script/Ciel/Day 9/09_01_CIEL09_6C.txt [hunk 5]
```diff
@@ -151,12 +151,12 @@
...
 -- Page 665, Offset 23862. Mods: @k, @e.
 -- 「――――――、
-"――――――,
+"――――――
 }
 [sha:c267ed739be5106eb77f21d065b9b536b48f3c2b]{
 -- Page 665, Offset 23863. Glued. Mods: @x.
 -- ア」
--- TRANSLATION HERE
+Ah.
```
Comment (rschlaikjer) [inline script/Ciel/Day 9/09_01_CIEL09_6C.txt:159]: Missing " how is this not a lint error

File: script/Ciel/Day 9/09_01_CIEL09_4.txt (hunk 4 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 9/09_01_CIEL09_4.txt:645]: Thinking about it, I added the offsets on the wrong files.

File: script/Ciel/Day 9/09_01_CIEL09_6C.txt (hunk 5 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 9/09_01_CIEL09_6C.txt:159]: my bad

**Pull request merged**